### ISSUE TITLE
Client gets disconnected on bad command

### ISSUE THREAD

**Issue by cmouse** [PowerDNS/weakforced#105]
Product: weakforced
Version: 1:1.2.0-2

```
~# wforce  -c
Read configuration from '/etc/wforce.conf'
Connecting to 0.0.0.0:4004
> status
Fatal error: EOF while writing message
````

And logs show
```
Jan 11 15:44:19 dev02 wforce[29825]: Got control connection from 127.0.0.1:58726
Jan 11 15:44:22 dev02 wforce[29825]: Got an exception in client connection from 127.0.0.1:58726: [string "status"]:1: '=' expected near '<eof>'
Jan 11 15:44:22 dev02 wforce[29855]: Fatal error: EOF while writing message
```

**Comment by neilcook** [comment]
Testing this I noticed a bunch of other issues regarding the console, so I will look at giving this code some love before the next release.

**Comment by neilcook** [comment]
Actually it turns out this is fixed in 00e2a8c, which means it will be part of 1.2.1.

While making the nonce-merging modifications, I noticed that dnsdist was catching some exceptions that wforce was not.
